src/metpy/plots/declarative.py

```diff
@@ -1043,9 +1043,12 @@ def _cmap_obj(self):
     def _norm_obj(self):
         """Return the normalization object.
 
-        Converts the tuple image range to a matplotlib normalization instance.
+        If `image_range` is a matplotlib normalization instance, returns it, otherwise converts 
+        the tuple image range to a matplotlib normalization instance.
 
         """
+        if isinstance(self.image_range, plt.Normalize):
+            return self.image_range
         return plt.Normalize(*self.image_range)
```

Comment:
Could we also get a second image test to cover the if statement and feeding a `plt.Normalize` for the `image_range` attribute? This will ensure that we have the ability to use that as intended since that is not currently covered by the test suite.